doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -310,11 +302,12 @@ Restart CORTX
    
 Tested by:
 
+- Mar 25, 2021: Mukul Malhotra (mukul.malhotra@seagate.com) using OVA release 1.0.2 & 1.0.3 on Windows 10 running Oracle VirtualBox & VMware Workstation.
 - Mar 24, 2021:  Harrison Seow (harrison.seow@seagate.com) using OVA release 1.0.2 on Windows running Oracle VM VirtualBox 6.1.16.
 
 - Mar 18, 2021: Jalen Kan (jalen.j.kan@seagate.com) using OVA release 1.0.2 on a Windows laptop running VMWare Workstation.
 
-- Feb 4, 2021:  Tim Coulter (timothy.r.coulter@seagate.com) using OVA release 1.0.2 on MAC running VMWare Fusion 12.1.0
+- Mar 18, 2021: Jalen Kan (jalen.j.kan@seagate.com) using OVA release 1.0.2 on a Windows laptop running VMWare Workstation.
```

Comment:
Duplicate entry for `Jalen Kan` and entry for `Tim Coulter` is removed. Please fix

Reply:
Tim Coulter entry is removed which is add conflict due to VMware Fusion

Reply:
done